Packs/FeedDHS/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### DHS Feed v2
+- Added support for TAXII V2 for the DHS feed.
```

Comment:
It's a new integration, so please add proper release notes.

Reply:
Updated